Makefile

```diff
@@ -124,16 +117,28 @@ clean-tmp: ## Removes temporary files
 	-@ $(foreach dir,$(TMP),rm -rf $(dir);)
 
 run: ## Starts containers and runs command $(CMD) inside the container in a non-interactive shell
-run: COMPOSE_FILE = $(COMPOSE_TEST_FILE)
-run: $(DOCKER_COMPOSE)
-	@echo
+run: MASTER_PASSWORD ?= "p"
+run: USER_PASSWORD ?= "p"
+run:
 	@echo "======= Run ======="
 	@echo
-	$(DOCKER_COMPOSE) run --rm --name $(PROJECT)-build-run $(DOCKER_ENV) build bash -c "$(CMD)"
+	@docker-compose run -e MASTER_PASSWORD=$(MASTER_PASSWORD) -e USER_PASSWORD=$(USER_PASSWORD) --rm system $(CMD)
+
+dev-setup: ## Makes the initial setup for the application ##
+dev-setup: CMD=rake db:setup
```

Comment:
How can I undo this?

Reply:
@josemigallas what do you mean by “undo” and what are your expectations?